### PR TITLE
[TDP] Remove the top border on the TDP Activity Index page

### PR DESCRIPTION
**Pull request by nbucknor**
Since the Notification block has been removed from the TDP Activity Search page, the top border is no longer necessary between the hero and the the search form.

---

## Removals

- Removed the top border from the search block on the TDP Activity Index page.

### Before:

Screen Shot 2020-08-24 at 1 23 22 PM

### After:

Screen Shot 2020-08-24 at 1 37 10 PM

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

File: cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
                 {%- endfor %}
             </div>
         </section>
-        <div class="block block__padded-top block__border-top u-mt0 u-mb30">
+        <div class="block block__padded-top u-mt0 u-mb30">
```
Comment (anselmbradford) [inline cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html:50]: While in here, you could update `u-mt0` to be `block__flush-top`, which would be the more "standard" way of removing the top margin.

Comment (nbucknor) [reply cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_index_page.html]: Done.

**Comment by nbucknor** [comment]
@Scotchester When is the earliest this can be merged?

**Comment by Scotchester** [comment]
> @Scotchester When is the earliest this can be merged?

@nbucknor As soon as it gets updated with master!

**Comment by nbucknor** [comment]
> > @Scotchester When is the earliest this can be merged?
> 
> @nbucknor As soon as it gets updated with master!

Oops. I've merged the latest version of master into my feature branch.

**Comment by Scotchester** [comment]
> Oops. I've merged the latest version of master into my feature branch.

Great, I'll merge as soon as the checks pass :)

**Pull request merged**